src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,28 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+export const MoviesList = ({ moviesList }) => (
+  <div className="movies">
+    {moviesList.map(movieItem => (
+      <div className="card" key={movieItem.imdbId}>
```

Comment:
As far as I see, you don't style `card` class. You can remove it and move `div` element inside `MovieCard` component.